docs/examples/Acme/Bundle/CustomBundle/AcmeCustomBundle.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Acme\Bundle\CustomBundle;
+
+use Symfony\Component\HttpKernel\Bundle\Bundle;
```

Comment:
Maybe all this AcmeBundle can be removed since we have the `custom-entity-demo` repo ?

Reply:
i would say that this is the custom-entity-demo that will be deprecated.
no need to have two repos to maintain in parallel.